docs/getting_started/howto/esmda_restart.rst

```diff
@@ -0,0 +1,28 @@
+Restarting ES-MDA
+-----------------
+Imagine you are conducting an experiment consisting of three iterations of ES-MDA, 
+with each iteration potentially taking hours or even days.
+For the purposes of this example, assume that the "Relative weights" are set to ``4, 2, 1``.
+
+As part of running this experiment, ERT creates four ensembles, which by default are named ``default_0``, ``default_1`` and ``default_2``, ``default_3``.
+Sadly, too many realizations fail during the model evaluation in ``default_3``, leaving no responses to analyse.
+Such failures can arise for various reasons, such as licensing problems with the simulator, unphysical parameterization etc.
```

Comment:
But if the parameterization is unphysical, a restart will not help, so perhaps add some information about which cases are worth restarting from?

Reply:
Thanks 👍 
I have also updated the sentence that started with "Sadly".